p2p/src/authenticated/mod.rs

```diff
@@ -158,7 +159,7 @@
 //!     let (sender, receiver) = network.register(
 //!         0,
 //!         Quota::per_second(NonZeroU32::new(1).unwrap()),
-//!         1024, // max message size
+//!         max_message_size,
```

Comment:
aside: If we weren't planning to remove chunking from `p2p`, I'd recommend making this some value > `max_message_size`.